File: SEM-1/Python/car game.py
print('''Welcome to car game
instructions are- help,start, stop, quit''')
n=""
started=False
while True:
    n = input("> ").lower()
    if n=="start":
        if started:
            print("car is already started!!!")
        else:
            started=True
            print("car started.....")
    elif n=="help":
        print("start - to start the car")
        print("stop - to stop the car")
        print("quit - to exit program")
    elif n=="stop":
        if not started:
            print("car is atready stopped.")
        else:
            started= False
            print("car is stopped!!!!!!")
    elif n=="quit":
        break
    else:
        print("I dont understand :-{(")

File: SEM-1/C-codes/problem/cargame.c
#include<stdio.h>
#include<string.h> 
int main()
{   int i=0;
    char s[20];
    printf("Welcome to my car game:\n");
    printf("Instruction are:\nhelp=get instructions.");
    while(i>0)
    {
        printf("Enter you order:");
        scanf("%s",s);
        if(s=='start')
        {
            printf("Car started moving.");
        }
        else if(s=='stop')
        {
            printf("Car stopped.");
        }
        else if(s=='help')
        {
            printf("\nstart=start the car.\nstop=stop the car.\nquit=quit the game.");
        }
        else
        {
            printf("I dont understand.");
        }
    }
}